test/images/agnhost/liveness/server.go

```diff
@@ -54,6 +60,13 @@ func main(cmd *cobra.Command, args []string) {
 			w.WriteHeader(200)
 			w.Write([]byte("ok"))
 		}
+		if disconnect {
+			if c, _, err := w.(http.Hijacker).Hijack(); err == nil {
+				c.Close()
```

Comment:
by this time, would headers be already written out? Or buffered? Or this depends on the buffer size?

Reply:
if the connection is hijacked the c will contain the raw tcp stream ... I can't remember but sounds unlikely the headers or any http protocol dependency gets buffered

Reply:
I added `Flush()` to send headers certainly to probe clients.